os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/inc/bt_board.h

```diff
@@ -79,7 +79,11 @@ extern "C" {
 #define EFUSE_SW_DRIVER_DEBUG_LOG    BIT6
 #define EFUSE_SW_RSVD2               BIT7
 
+#ifdef CONFIG_BLE_DEBUG
 #define CHECK_SW(x)                  (HAL_READ32(SPI_FLASH_BASE, FLASH_SYSTEM_DATA_ADDR + 0x28) & x)
+#else
+#define CHECK_SW(x)                  1
```

Comment:
Why do you remain 1 when `CONFIG_BLE_DEBUG` is not enabled?
How about remaining empty instead of 1?

Reply:
Other will call this CHECK_SW(x) and expect a return value, 0 mean enable, 1 disable.
So it have to return as 1 to disable all debug function.